core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -572,36 +572,19 @@ public enum Property {
       "Time a tablet server will sleep between checking which tablets need compaction.", "1.3.5"),
   TSERV_COMPACTION_SERVICE_PREFIX("tserver.compaction.major.service.", null, PropertyType.PREFIX,
       "Prefix for compaction services.", "2.1.0"),
-  TSERV_COMPACTION_SERVICE_ROOT_PLANNER("tserver.compaction.major.service.root.planner",
```

Comment:
Are these properties being deleted by this change?  (It's hard to see with the UI diff) - if so, then we need to make sure that there is a corresponding deprecation marking in an earlier release - or potentially be able to map from old to new names in this version.

Reply:
Yes, two sets of Properties are being collapsed into a single one. 

These specific properties are a bit odd since they are service definitions. So the code is looking for any properties that match`tserver.compaction.major.service` and then selecting the next property segment as the service name and performing property lookup operations using that service name. 

if a user wants to specify a new compaction service, they add their own property name `tserver.compaction.major.service.user` and then add properties under that property name 
```
   tserver.compaction.major.service.user
   tserver.compaction.major.service.user.planner
   tserver.compaction.major.service.user.rate.limit
   tserver.compaction.major.service.user.planner.opts.prop1
```

The properties in question are only being used by the accumulo `init/InitialConfiguration` which has been updated to look for the new compaction service name. 

Overall I think there are still some outstanding organization changes with compaction code that may change these property names further. I can add the deprecation annotations now, but I'm not sure how we want to track a large scale property name refactor like say, removing the tserver prefix from `tserver.compaction`.

Reply:
My comment was more of a prompt that it may be something that we need to keep in mind - not sure of the best approach at this point, but if it could impact user configs, then we need to keep that in mind.

Reply:
In `elasticity` compactions no longer occur in the tserver. I wonder if the appropriate action here is to deprecate the existing properties and use a different prefix. Maybe under `general`?